package.json

```diff
@@ -55,7 +56,7 @@
   "license": "MIT",
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "npx pretty-quick --staged"
```

Comment:
Why `npx`, AFAIK scripts will use local packages _anyways_, although I see that `npx` will:

> installing any packages needed in order for <command> to run

Is there some general "Thou shalt use npx" dictate / reasoning?

Reply:
Do we always need/want latest?

Reply:
Husky says to do it this way. See: https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick

I don't know why though. :thinking:

Reply:
I wonder if it is so it still works if you didn't `npm install` before committing? If that is the reason, it should probably be `npx pretty-quick@3.1.3 ...`. BUTTTT if pretty-quick isn't installed then `prettier` probably isn't either... so it'd probably fail anyway. :thinking:  :thinking:

Reply:
Anyone have opinions on what we should do here? Probably need to pin the dependency version and we'll be set? Or should we go against the recommendation?

Reply:
pretty-quick's own docs say to do

```
"husky": {
  "hooks": {
    "pre-commit": "pretty-quick --staged"
  }
}
```

https://github.com/azz/pretty-quick#pre-commit-hook

hm

Reply:
I think I'm more comfortable _without_ `npx`.

Reply:
I think I would side with the `pretty-quick` docs over `husky` on that.